modules/snooker/src/main/ts/ephox/snooker/api/Warehouse.ts

```diff
@@ -47,45 +48,53 @@ const generate = function <T extends Structs.Detail> (list: Structs.RowData<T>[]
   //          rowspan (merge cols)
   const access: Record<string, Structs.DetailExt> = {};
   const cells: Structs.RowData<Structs.DetailExt>[] = [];
+  let groups: SugarElement[] = [];
```

Comment:
You really should be capturing the `colspan` here as well, so this should probably be a struct rather than a dom element. That will also keep it consistent with how that's accessed for cells.